pkg/db/db_local/service.go

```diff
@@ -22,9 +22,9 @@ func GetState() (*RunningDBInstanceInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+	log.Printf("[TRACE] GetState - loadRunningInstanceInfo returned %#v\n", info)
 
 	if info == nil {
-		log.Println("[TRACE] GetRunStatus - loadRunningInstanceInfo returned nil ")
```

Comment:
Let's keep this line. This should not happen under normal circumstance - but when it does, it'll help immensely if we see this line in the log files.